eZ/Publish/Core/SignalSlot/Signal/RoleService/UnassignRoleByAssignmentSignal.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * UnassignRoleByAssignmentSignal class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace eZ\Publish\Core\SignalSlot\Signal\RoleService;
+
+use eZ\Publish\Core\SignalSlot\Signal;
+
+/**
+ * UnassignRoleByAssignmentSignal class.
+ */
+class UnassignRoleByAssignmentSignal extends Signal
```

Comment:
This can be renamed to be in sync with method rename: `RemoveRoleAssignmentSignal`.